<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CrashUploadDialog</class>
 <widget class="QDialog" name="CrashUploadDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>147</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>A crash has been detected while running RealGUIDE. Would you like to report this issue to the development team?</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxProject">
     <property name="text">
      <string>Include the project files</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnNever">
       <property name="text">
        <string>Never</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnOk">
       <property name="text">
        <string>Once</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAlways">
       <property name="text">
        <string>Always</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
